src/components/Layout/index.jsx

```diff
@@ -0,0 +1,144 @@
+import React, { useState } from "react";
+import "./style.scss";
+import { Container } from "react-bootstrap";
+import { BsStar, BsStarFill } from "react-icons/bs";
+
+import Header from "../Header";
+import Treeview from "./components/Treeview";
+
+const typeList = [
+  "Trend cases",
+  "Ult protection case",
+  "Ink cartridges",
+  "Bussiness cases",
+  "Connectivity",
+];
+const brandList = ["Samsung", "Metra", "HP", "Apple"];
+const priceRangeList = [
+  "< 1",
+  "1 - 80",
+  "80 - 160",
+  "80 - 160",
+  "160 - 240",
+  "240 - 1.820",
+  "1.820 - 3.400",
+  "3.400 - 4.980",
+  "> 4.980",
+];
+const stars = [1, 2, 3, 4];
+
+export default function Layout(props) {
+  const renderRefindType = () => {
+    return (
+      <div className='refineby-type'>
+        <h5>Type</h5>
+        {typeList.map((type) => {
+          return (
+            <label key={type}>
+              <input type='checkbox' />
+              {type}
+            </label>
+          );
+        })}
```

Comment:
```suggestion
        {typeList.map((type) => (
            <label key={type}>
              <input type='checkbox' />
              {type}
            </label>
          ))}
```
Mấy cái return đơn giản như này thì k cần keyword return với {} đâu anh